app/javascript/packs/verify-flow.tsx

```diff
@@ -44,6 +49,15 @@ const {
 const initialValues = JSON.parse(initialValuesJSON);
 const enabledStepNames = JSON.parse(enabledStepNamesJSON) as string[];
 
+const camelCase = (string: string) =>
+  string.replace(/[^a-z]([a-z])/gi, (_match, nextLetter) => nextLetter.toUpperCase());
+
+const jwtData = JSON.parse(atob(initialValues.userBundleToken.split('.')[1]));
+const pii = Object.fromEntries(
+  Object.entries(jwtData.pii).map(([key, value]) => [camelCase(key), value]),
+);
```

Comment:
As discussed in our working session, really tempting to pull in [Lodash](https://lodash.com/) here:

```suggestion
import { mapKeys, camelCase } from 'lodash-es';

const jwtData = JSON.parse(atob(initialValues.userBundleToken.split('.')[1]));
const pii = mapKeys(jwtData.pii, camelCase);
```